cmd/kubeadm/app/cmd/token.go

```diff
@@ -247,7 +247,7 @@ func RunCreateToken(out io.Writer, client clientset.Interface, cfgPath string, i
 	// In fact, we don't do any CRI ops at all.
 	// This is just to force skipping the CRI detection.
 	// Ref: https://github.com/kubernetes/kubeadm/issues/1559
-	initCfg.NodeRegistration.CRISocket = kubeadmconstants.DefaultDockerCRISocket
+	initCfg.NodeRegistration.CRISocket = kubeadmconstants.UnknownCRISocket
```

Comment:
this seems to apply to all commands in kubeadm that have configuration input but no actual container runtime interaction?

we added this fix for `token` but i can think of a few other commands that may need this too, e.g. `kubeadm kubeconfig` or `kubeadm certs`.

Reply:
@neolit123  Let me check other commands today.

BTW, is `unknown` a good naming here? It is something like `no need to detect or not detected yet`.

Reply:
`unknown` seems fine for the time being.